build.gradle

```diff
@@ -28,3 +28,4 @@ allprojects {
     }
 }
 
+apply plugin: 'kotlin-android-extensions'
```

Comment:
Kotlin plugin is already applied in build.gradle (app module).

Reply:
But is giving warnings when compiled. Giving unresolved reference error for kotlinx. After adding this all the warnings are gone.

Reply:
There are no such warnings on the compilation on my side.  Do check again.

Reply:
Yaa, may be it is not more, after installing the compiler.